components/Article/EditLink/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -43,18 +41,16 @@ exports[`EditLink component Translate renders correctly 1`] = `
         Interested to help with translations? (Non-English)
       </span>
       <svg
-        aria-hidden="true"
-        class="svg-inline--fa fa-pencil "
-        data-icon="pencil"
-        data-prefix="fas"
-        focusable="false"
-        role="img"
+        fill="currentColor"
+        height="1em"
+        stroke="currentColor"
+        stroke-width="0"
         viewBox="0 0 512 512"
+        width="1em"
         xmlns="http://www.w3.org/2000/svg"
       >
         <path
-          d="M410.3 231l11.3-11.3-33.9-33.9-62.1-62.1L291.7 89.8l-11.3 11.3-22.6 22.6L58.6 322.9c-10.4 10.4-18 23.3-22.2 37.4L1 480.7c-2.5 8.4-.2 17.5 6.1 23.7s15.3 8.5 23.7 6.1l120.3-35.4c14.1-4.2 27-11.8 37.4-22.2L387.7 253.7 410.3 231zM160 399.4l-9.1 22.7c-4 3.1-8.5 5.4-13.3 6.9L59.4 452l23-78.1c1.4-4.9 3.8-9.4 6.9-13.3l22.7-9.1v32c0 8.8 7.2 16 16 16h32zM362.7 18.7L348.3 33.2 325.7 55.8 314.3 67.1l33.9 33.9 62.1 62.1 33.9 33.9 11.3-11.3 22.6-22.6 14.5-14.5c25-25 25-65.5 0-90.5L453.3 18.7c-25-25-65.5-25-90.5 0zm-47.4 168l-144 144c-6.2 6.2-16.4 6.2-22.6 0s-6.2-16.4 0-22.6l144-144c6.2-6.2 16.4-6.2 22.6 0s6.2 16.4 0 22.6z"
-          fill="currentColor"
+          d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"
```

Comment:
Would be great if you could add a screenshot for this

Reply:
react-icons:
![Screenshot 2023-04-20 at 8 25 13 PM](https://user-images.githubusercontent.com/128208841/233365268-7af5d1cd-7cd5-43e0-8cc2-04c0a942ae3b.png)

Font Awesome:
https://fontawesome.com/v4/icon/pencil

How it looks like, you can see screenshots below.